### ISSUE TITLE
s07_describe - DUE 9/6

### ISSUE THREAD

**Issue by jessfroe** [lzim/teampsd#506]
# s07 CHECKLIST

## 1. Link to the document and add relevant team members to the card.
- [x] Add the link the current draft of the documentation for the issue: https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/learner_see/mtl_session07_see.md

## 2. Link to issues that must be edited in the document in the **issue_tracker** and **feature_tracker.** _(use the # for the link to the issue)_
- [n/a ] Issue #451 
- [ ] Issue # 
- [ ] Issue # 

## 3. Team Review
- [ ] Fidelity - check with Lindsey
- [ ] Data UI - check quant_workgroup
- [ ] Sim UI - check with model_workgroup and/or sim_workgroup

## 4. Review and Open Pull Requests _(use the # for the link to the pull request)_ #635 
- [x] Check the draft for typos
- [x] Check or open a pull request for the document edits

## 5. Review and/or Update Graphics
- [x] Screenshots
- [x] Gifs

- [x] 1. Blue section bar title change from Outputs -> Outputs and Text.
- [x] 2. The Step 1 gif only shows typing the mtl.how/sim into a Chrome browser - do we have one that includes logging in?
- [x] 3. Step 3 gif should mouse over Experiment sliders to show they're set to 0 or bc. It does not need to circle the Team Data Table.
- [x] 4. Step 4 gif should show only typing in the q/h boxes (not f/d), and should prime those by showing: "What will it mean for our team over the next 2 years if we make no new decisions?" and "If we make no new decisions in our team, we expect to see ___ over the next two years."
- [ ] 5. gifs for some of the remaining steps could be helpful; but are not required and may be more of a visual change (for videos) than desired.

- [x] Logos/Icons
- [x] Links

## 6. Dependencies
- [ ] Make sure that any edits you make that need to be added to **depend_product** are tracked (e.g., _MTL Video_)
_I don't think these changes require any changes to videos. The change from "Outputs" to "Outputs and Text" should be noted in **S05 video**_
_Note to all facilitators: I think our nomenclature should be "Text section" and "Outputs and Text section", not "Expanded Outputs section" or "Expanded Outputs and Text section". Just use the name in the blue bar. You can say to "expand" the Outputs and Text section (by using the expand/collapse icon) but it's not part of the name._

## 7. Close Pull Requests
- [ ] Request review and/or close open pull requests for the document

**Comment by staceypark** [comment]
@branscombj @dlkibbe 

I created a new gif for Step 3 according to this change. The new gif I uploaded is still called the "sim_ui_sections" like before so it changed in all instances of this gif.
>1. Blue section bar title change from Outputs -> Outputs and Text.

I already created an updated gif called "sim_ui_login.gif" a while back to address this. It looks like I hadn't updated it everywhere though. In my most recent pull request, I updated this gif under step 1. You can always look at https://github.com/lzim/teampsd/tree/master/resources/gifs for the full list of gifs.

> 2. The Step 1 gif only shows typing the mtl.how/sim into a Chrome browser - do we have one that includes logging in?

I'm still working on updating the gif guide & associated workflow decisions that you need to be aware of. I will alert you when I've finished it.

**Comment by staceypark** [comment]
@branscombj @dlkibbe I've updated the gif guide (https://github.com/lzim/teampsd/tree/master/resources/gif_guide) to include step by step instructions on recording your screen using either PowerPoint or Screenomatic and converting to gifs using ezgif. 

Note: As mentioned in the gif guide, when updating existing gifs, you want to keep the ***same name*** for the file name. This way when you delete the old one and update the new one, it will automatically update across all files that use that gif.

As I mentioned above, I updated the first 2 gif requests. However, based on comment 5 (below) it may be faster and clearer for you to update the gifs as you're working on the guides, since the changes that need to be made will be clearest to you.

> 5. gifs for some of the remaining steps could be helpful; but are not required and may be more of a visual change (for videos) than desired.
